hpx/runtime/threads/executors/thread_pool_executors.hpp

```diff
@@ -104,6 +107,13 @@ namespace hpx { namespace threads { namespace executors
 
             // resource manager registration
             std::size_t cookie_;
+
+            // store the self reference to the HPX running this scheduler
```

Comment:
Yes, thanks. I have that fixed already. There are more things to clean up from the latest test results.